lighthouse-core/report/html/renderer/performance-category-renderer.js

```diff
@@ -165,6 +165,20 @@ class PerformanceCategoryRenderer extends CategoryRenderer {
       filmstripEl && timelineEl.appendChild(filmstripEl);
     }
 
+    // Budgets
+    const budgetAudit = category.auditRefs.find((audit) => audit.id === 'performance-budget');
```

Comment:
also not sure if we should do this by audit id or by group and/or if there's any point to having the `budgets` group anymore if we keep it by id?

FWIW I think I'd lean towards group since that's how we're doing everything else, but then again it might be a bit confusing that it's a single audit in the group so 🤷‍♂